src/integrate/orchestrator.rs

```diff
@@ -675,7 +677,7 @@ rpcport={}
         bitcoind_data_path.push(format!("{}", boot_index));
         let _ = fs::create_dir(bitcoind_data_path.clone());
         bitcoind_data_path.push("bitcoin");
-        fs::create_dir(bitcoind_data_path).expect("Unable to create working dir");
+        let _ = fs::create_dir(bitcoind_data_path);
```

Comment:
I have the same question as below with these new `expect`s. If the function returns a `Result`, then shouldn't this error be returned instead of panicking? Otherwise, why bother returning a `Result`?